includes/admin/tools/export/give-export-donations-exporter.php

```diff
@@ -365,6 +368,11 @@ public function get_data() {
 					$data[ $i ]['address_country'] = isset( $address['country'] ) ? $address['country'] : '';
 				}
 
+				if ( ! empty( $columns['comment'] ) ) {
+					$comment               = give_get_donor_donation_comment( $payment->ID, $payment->donor_id );
+					$data[ $i ]['comment'] = $comment->comment_content;
```

Comment:
@DevinWalker comment can empty array to validate properly before accessing `comment_content`.

fixed: https://github.com/impress-org/give/commit/cf3f8271035dbed496312c5cbc7718a79bd7210f